_includes/headertop.html

```diff
@@ -12,6 +12,16 @@
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
+    <meta property="og:site_name" content="{{ site.title }}"/>
+    {% if page.title %}<meta property="og:title" content="{{ page.title }}"/>{% endif %}
+    {% if page.description %}<meta property="og:description" content="{{ page.description }}"{% endif %}
```

Comment:
Should we also provide the `og:type`, `og:image`, and `og:url` properties? They are documented as **required**.

https://ogp.me/